server/package.json

```diff
@@ -4,56 +4,30 @@
     "private": true,
     "homepage": "/tiltaksgjennomforing",
     "scripts": {
-        "build": "cross-env GIT_COMMIT_HASH=$(git log -n 1 --pretty=format:'%h') webpack --config webpack.config.js",
-        "release": "cross-env GIT_COMMIT_HASH=$(git log -n 1 --pretty=format:'%h') node sentry.js",
-        "server": "node server.js --openssl-legacy-provider ",
-        "start": "node ../dist/server.js --openssl-legacy-provider",
-        "start-ekstern": "cross-env ENABLE_EXTERNAL_MENU=true yarn start",
-        "start-intern": "cross-env process.env.ENABLE_INTERNAL_MENU=true yarn start",
-        "test": "cross-env CI=true yarn test -w 1",
-        "test:config": "node -e 'console.dir(require(\"./jest.config.js\"))'",
-        "test:help": "yarn test --help",
-        "up": "docker-compose up -d --remove-orphans --build"
+        "build": "esbuild src/server.ts --bundle --platform=node --outfile=../dist/server/index.cjs --packages=external",
+        "postbuild": "cp -r node_modules ../dist/server",
+        "start": "node ../dist/server/index.cjs --openssl-legacy-provider",
+        "start-ekstern": "cross-env ENABLE_EXTERNAL_MENU=true npm start",
+        "start-intern": "cross-env process.env.ENABLE_INTERNAL_MENU=true npm start"
     },
     "dependencies": {
         "@navikt/arbeidsgiver-notifikasjoner-brukerapi-mock": "^6.2.2",
```

Comment:
ny versjon 6.5.0 av `@navikt/arbeidsgiver-notifikasjoner-brukerapi-mock` er tilgjengelig

Reply:
Vi er på 6.5.0 - du må se i `package-lock` for å finne faktisk versjon. Siden vi har `^` så reflekteres ikke siste minor/patch i `package.json`